DATA/production/configurations/2022/LHC22f/apass1/async_pass.sh

```diff
@@ -366,6 +366,9 @@ else
   #
   if [[ "0$ASYNC_PASS_NO_OPTIMIZED_DEFAULTS" != "01" ]]; then
     export TIMEFRAME_RATE_LIMIT=3
+    if (( $(echo "$RUN_IR > 800000" | bc -l) )); then
```

Comment:
What is the content of $RUN_IR? Is is a complicated string, or just an integer?

Reply:
The IR is a float that we extract from CCDB at the very beginning, it is here:
https://github.com/AliceO2Group/O2DPG/blob/99361c5fcf9c2db3bf97a57c57c7bfae78aaa1eb/DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh#L149-L155